docs/security/authentication/index.asciidoc

```diff
@@ -80,32 +81,93 @@ IMPORTANT: The {kib} user-facing origin should be the same in {kib}, {es}, and t
 
-During logout, both the {kib} session cookie and access/refresh token pair are invalidated. Even if the cookie has been leaked, it can't be re-used after logout. This is known as SAML "local" logout.
+[[oidc]]
+==== OpenID Connect Single Sign-On
+
+Similar to SAML, authentication with OpenID Connect allows users to log in to {kib} using an OpenID Connect Provider such as Google, or Okta. OpenID Connect
+should also be configured in {es}, see {xpack-ref}/saml-guide.html[Configuring OpenID Connect Single-Sign-On on the Elastic Stack] for more details.
+
+Set the configuration values in `kibana.yml` as follows:
+
+. Enable the SAML authentication:
++
+[source,yaml]
+--------------------------------------------------------------------------------
+xpack.security.authProviders: [oidc]
+--------------------------------------------------------------------------------
+
+. {kib} needs to specify which OpenID Connect realm in {es} should be used, in case there are more than one configured there.
++
+[source,yaml]
+--------------------------------------------------------------------------------
+xpack.security.autc.oidc.realm: oidc1
+--------------------------------------------------------------------------------
 
-{kib} can also initiate a SAML "global" logout or _Single Logout_ if it's supported by the Identity Provider and not explicitly disabled by {es}. In this case, the user is redirected to the Identity Provider for log out of all applications associated with the active Identity Provider session.
+. {kib} supports Third Party initiated Single Sign On, which might start with an external application instructing the user's 
+browser to perform a "non-safe" `POST` HTTP method. This request will not include CSRF protection HTTP headers that are 
+required by {kib}. If you want to use Third Party initiated SSO , then you must disable the CSRF check for this endpoint.
++
+[source,yaml]
+--------------------------------------------------------------------------------
+server.xsrf.whitelist: [/api/security/v1/oidc]
+--------------------------------------------------------------------------------
 
 [float]
-===== SAML and Basic Authentication
+===== OpenID Connect and Basic Authentication
 
-SAML support in {kib} is designed to be the primary (or sole) authentication method for users of that {kib} instance. However, you can configure both SAML and Basic authentication for the same {kib} instance:
+Similar to SAML, OpenID Connect support in {kib} is designed to be the primary (or sole) authentication method for users 
+of that {kib} instance. However, you can configure both OpenID Connect and Basic authentication for the same {kib} instance:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-xpack.security.authProviders: [saml, basic]
+xpack.security.authProviders: [oidc, basic]
 --------------------------------------------------------------------------------
 --
```

Comment:
```suggestion
```
I don't know how it worked previously, but I'm unable to build the docs using `docbldkb` from https://github.com/elastic/docs/blob/master/doc_build_aliases.sh#L26 without deleting this line... 🤷‍♂